install/kubernetes/microcks/values.yaml

```diff
@@ -54,6 +54,12 @@ microcks:
 postman:
   image: quay.io/microcks/microcks-postman-runtime:latest
   replicas: 1
+  resources:
+    requests:
+      cpu: 30m
```

Comment:
`30m` seems a bit low for this container. Do you know if this value is based on measured experiments? Some users in the community are using `postman-runtime` intensively, and I think this low value will impact them... Could you please stick to the previous version? (actually not specifying `request.cpu`). You'll still be able to include this value in your own.

Reply:
good catch, I totally made up this one.
So I'll remove it from the default value